app/models/user.rb

```diff
@@ -34,6 +34,10 @@ class User < ActiveRecord::Base
       waiting_for_email: 'waiting for email confirmation'
   }
 
+  after_create :notify_admin
```

Comment:
maybe `after_commit :notify_admin, on: :create` is better? So you are sure the user is persisted